api/server/server_windows.go

```diff
@@ -0,0 +1,19 @@
+// +build windows
+
+package server
+
+import (
+	"fmt"
+	"github.com/docker/docker/engine"
+)
+
+// NewServer sets up the required Server and does protocol specific checking.
+func NewServer(proto, addr string, job *engine.Job) (Server, error) {
+	// Basic error and sanity checking
+	switch proto {
+	case "tcp":
+		return setupTcpHttp(addr, job)
+	default:
+		return nil, fmt.Errorf("Invalid protocol format. Windows only supports tcp.")
```

Comment:
"protocol format" or just "protocol"? Also this is an Errorf without a %s. :smile: I suggest making this:

> `Invalid protocol %s. Windows only supports tcp.`